packages/theming/src/index.ts

```diff
@@ -13,4 +13,5 @@ export { default as retrieveComponentStyles } from './utils/retrieveComponentSty
 export { default as withTheme } from './utils/withTheme';
 export { default as getDocument } from './utils/getDocument';
 export { default as getColor } from './utils/getColor';
+export { default as arrowStyles, POSITION as ARROW_POSITION } from './utils/arrowStyles';
```

Comment:
If we're exporting this as `ARROW_POSITION` could we use that naming internally? It seems to make sense in the `arrowStyles` utility